testsuite/single_node_performance.py

```diff
@@ -136,6 +137,11 @@ class RunGroupConfig:
     RunGroupConfig(expected_tps=1300, key=RunGroupKey("token-v1nft-mint-and-transfer-parallel"), included_in=Flow(0)),
     RunGroupConfig(expected_tps=5300, key=RunGroupKey("token-v1nft-mint-and-transfer-parallel", module_working_set_size=20), included_in=Flow(0)),
 
+    RunGroupConfig(expected_tps=14000, key=RunGroupKey("coin-init-and-mint", module_working_set_size=1), included_in=LAND_BLOCKING_AND_C),
```

Comment:
LAND_BLOCKING_AND_C not defined?

Reply:
it's defined as:

LAND_BLOCKING_AND_C = Flow.LAND_BLOCKING | Flow.CONTINUOUS

Reply:
hmm.. where? :trollface:

Reply:
right below the enum - https://github.com/aptos-labs/aptos-core/blob/main/testsuite/single_node_performance.py#L37

Reply:
My IDE sucks (no, I suck, I searched in '*.rs')